crates/sui-e2e-tests/tests/rpc/client.rs

```diff
@@ -82,3 +83,35 @@ async fn get_full_checkpoint() {
         .await
         .unwrap();
 }
+
+#[sim_test]
+async fn get_checkpoint_artifacts() {
+    // TODO: remove this once artifacts digest is enabled on mainnet.
```

Comment:
tests use the unknown chain id, same as devnet, so this shouldn't be necessary

Reply:
I put this check in because it failed in simtest-mainnet on CI. See https://github.com/MystenLabs/sui/actions/runs/17596512638/job/49989884849

Reply:
I see - we should try to remember to remove that once the feature is on mainnet then